netmanager/src/redux/Join/actions.js

```diff
@@ -67,39 +67,46 @@ import {
   VERIFY_TOKEN_URI,
   UPDATE_PWD_IN_URI,
   REGISTER_CANDIDATE_URI,
-  DEFAULTS_URI } from "config/urls/authService";
-import { setDefaultAirQloud } from "../AirQloud/operations";
-
+  DEFAULTS_URI
+} from 'config/urls/authService';
+import { setDefaultAirQloud } from '../AirQloud/operations';
 
 /***************************errors ********************************* */
 
-export const clearErrors = () => dispatch => {
-  dispatch({ type: CLEAR_ERRORS })
-}
+export const clearErrors = () => (dispatch) => {
+  dispatch({ type: CLEAR_ERRORS });
+};
 
 /***************************organization actions ********************************* */
 export const setOrganization = () => (dispatch, getState) => {
-  const name = getState().auth.user.organization
-  dispatch(updateOrganization({ name }))
-}
+  const name = getState().auth.user.organization;
+  if (name) {
+    dispatch(updateOrganization({ name }));
+  } else {
+    dispatch(updateOrganization('airqo'));
```

Comment:
thanks @FaithDaka 

In the near future, we shall only be using one tenant (AirQo) in our system.
All other networks will be present under the AirQo tenant.

cc: @Codebmk